sdk/appconfiguration/azure-data-appconfiguration/src/test/java/com/azure/data/appconfiguration/ConfigurationClientBuilderTest.java

```diff
@@ -110,6 +110,7 @@ public void timeoutPolicy() {
             ? "Endpoint=http://localhost:8080;Id=0000000000000;Secret=MDAwMDAw"
             : Configuration.getGlobalConfiguration().get(AZURE_APPCONFIG_CONNECTION_STRING);
 
+        // What exactly is this testing?
```

Comment:
It tests if the `TimeoutPolicy" could be working in the client.

Reply:
Could this be removed for a test in `azure-core`?